torch/quantization/fx/quantize.py

```diff
@@ -981,7 +985,7 @@ def visit_arg(arg):
                     # overwrite previous activation post process constructor if
                     # necessary
                     quants[arg.name] = (
-                        DefaultQuantizeHandler(self, arg), act_post_process_ctr)
+                        DefaultQuantizeHandler(self, arg), act_post_process_ctr, is_weight)
```

Comment:
I think we don't need is_weight, this info is used to decide act_post_process_ctr already in L976

Reply:
also we can remove L978-979 I think